packages/expo-modules-autolinking/src/reactNativeConfig/reactNativeConfig.ts

```diff
@@ -196,3 +207,12 @@ export async function resolveAppProjectConfigAsync(
 
   return {};
 }
+
+/**
+ * Extracts the major version number from the 'expo' dependency string.
+ *
+ * @returns The major version number or 0.
+ */
+function getExpoVersion(packageJson: { dependencies?: { expo?: string } }): number {
+  return +(packageJson?.dependencies?.expo?.match(/\d+/)?.[0] ?? '0');
+}
```

Comment:
is there any other logic that we can reuse for this? for example, `@expo/config` resolves the version in the `getConfig()` function

Reply:
maybe reuse the `getConfig` from #35958